<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitHub Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="profileImageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>username</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bioLabel">
           <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
               </sizepolicy>
           </property>
           <property name="text">
               <string>bio</string>
           </property>
           <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
       </widget>
      </item>
         <item>
             <widget class="QPushButton" name="editProfileButton">
                 <property name="text">
                     <string>Edit profile</string>
                 </property>
             </widget>
         </item>
         <item>
             <widget class="IconLabel" name="companyWidget" native="true">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
             </widget>
         </item>
         <item>
             <widget class="IconLabel" name="locationWidget" native="true">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
             </widget>
         </item>
         <item>
       <widget class="IconLabel" name="twitterWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="IconLabel" name="joinedWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
         <item>
             <spacer name="verticalSpacer">
                 <property name="orientation">
                     <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                     <size>
                         <width>20</width>
                         <height>40</height>
                     </size>
                 </property>
             </spacer>
         </item>
         <item>
             <widget class="QPushButton" name="infoButton">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="text">
                     <string/>
                 </property>
             </widget>
         </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="mainTabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
            <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
           <property name="documentMode">
               <bool>false</bool>
           </property>
           <property name="tabsClosable">
               <bool>false</bool>
           </property>
           <property name="movable">
               <bool>false</bool>
           </property>
           <widget class="QWidget" name="overviewTab">
               <attribute name="icon">
                   <iconset>
                       <normaloff>../resources/icons/dark open book.png</normaloff>../resources/icons/dark open book.png
                   </iconset>
               </attribute>
               <attribute name="title">
                   <string>Overview</string>
               </attribute>
           </widget>
           <widget class="QWidget" name="repositoriesTab">
               <attribute name="icon">
                   <iconset>
                       <normaloff>../resources/icons/dark repo.png</normaloff>../resources/icons/dark repo.png
                   </iconset>
               </attribute>
               <attribute name="title">
                   <string>Repositories</string>
               </attribute>
           </widget>
           <widget class="QWidget" name="localTab">
               <attribute name="icon">
                   <iconset>
                       <normaloff>../resources/icons/dark star.png</normaloff>../resources/icons/dark star.png
                   </iconset>
               </attribute>
               <attribute name="title">
                   <string>Local</string>
               </attribute>
           </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
     <widget class="QMenuBar" name="menubar">
         <property name="geometry">
             <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>1020</width>
                 <height>22</height>
             </rect>
         </property>
         <widget class="QMenu" name="menuHelp">
             <property name="title">
                 <string>Help</string>
             </property>
             <addaction name="actionAyuda"/>
             <addaction name="separator"/>
             <addaction name="actionAbout"/>
         </widget>
         <widget class="QMenu" name="menuMenu">
             <property name="title">
                 <string>Menu</string>
             </property>
             <addaction name="actionReload"/>
             <addaction name="separator"/>
             <addaction name="actionExit"/>
         </widget>
         <addaction name="menuMenu"/>
         <addaction name="menuHelp"/>
     </widget>
     <widget class="QStatusBar" name="statusbar"/>
     <action name="actionAbout">
         <property name="text">
             <string>Acerca de</string>
         </property>
     </action>
     <action name="actionAyuda">
         <property name="text">
             <string>Ayuda de GitHub Manager</string>
         </property>
     </action>
     <action name="actionExit">
         <property name="text">
             <string>Exit</string>
         </property>
     </action>
     <action name="actionReload">
         <property name="text">
             <string>Reload</string>
         </property>
         <property name="shortcut">
             <string>Ctrl+R</string>
         </property>
     </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconLabel</class>
   <extends>QWidget</extends>
   <header>widgets.IconLabel</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
